gdal/frmts/derived/deriveddataset.cpp

```diff
@@ -37,8 +37,10 @@ class DerivedDataset : public VRTDataset
     public:
         DerivedDataset(int nXSize, int nYSize);
         ~DerivedDataset();
-
+  
+        static int Identify( GDALOpenInfo * );
```

Comment:
In the register function, you need to do poDriver->pfnIdentify = Identify, so that the method is actually used